File: Java/src/main/java/com/DSA/Java/LinkedList/Double/Runner.java
package com.DSA.Java.LinkedList.Double;

public class Runner {
   public static void main(String[] args) {
      DoublyLinkedList dLL = new DoublyLinkedList();
      dLL.createList(10);
      dLL.insertNode(5, 0);
      dLL.insertNode(15, 2);
      dLL.insertNode(20, 3);
      dLL.insertNode(25, 4);
      
      dLL.traverse();
      dLL.deleteList();

      // dLL.searchNode(15);
      // dLL.deleteNode(0);
      // dLL.traverse();

      // dLL.deleteNode(5);
      // dLL.traverse();

      // dLL.deleteNode(1);
      dLL.traverse();
   }
}

File: C++/LinkedList/Double/Runner.cpp
#include "DoublyLinkedList.h"
#include <iostream>

int main() {
   DoubleLinkedList dll;
   dll.insert_element(5, 0); 
   dll.insert_element(15, 1); 
   dll.insert_element(10, 1); 
   dll.insert_element(20, 2); 
   dll.insert_element(25, 5); 
   
   dll.traverse();

   dll.delete_element(5);
   dll.traverse();

   dll.delete_element(25);
   dll.traverse();

   dll.delete_element(10);
   dll.traverse();

   dll.delete_element(5);
   dll.delete_list();
   dll.traverse();

   return 0;
}
